projects/atomic-swap-service/headless-wallet.md

```diff
@@ -104,9 +104,9 @@ A new route `[DELETE] /wallet/atomic-swap/tx-proposal/delete/{proposalId}` will
 Also, should a wallet be stopped, all proposals it's currently listening to should also be removed, and its websocket connection closed.
 
 ### External notifications
-For every listened proposal a websocket connection will be opened with the Atomic Swap Service, informing the application about any changes that happen to it through the event `wallet:update-proposal`.
+For every wallet with a registered proposal, a websocket connection will be opened with the Atomic Swap Service, informing the application about any changes that happen to its proposals through the event `wallet:update-proposal`.
```

Comment:
I don't really like the name `wallet:update-proposal`. A proposal can reference a lot of things, in this case an atomic swap proposal.

I would use `wallet:update-atomic-swap-proposal`

Reply:
Good point! Added the description with the names of the events both from the wallet lib and the one emitted at the external notifications.